cmd/gonix/main_test.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestDrvShow tests the `go-nix drv show` command.
+func TestDrvShow(t *testing.T) {
+	drvs := []string {"../../test/testdata/7dsin4zpzi50wv7nia5nsi813nyrjgal-escape_html.drv"};
+	for _, drvBasename := range drvs {
+		os.Args = []string{"gonix", "drv", "show", ("../../test/testdata/" + drvBasename)}
```

Comment:
I'm not entirely sure we should invoke the binary here, or the handlers directly (`cmd/gonix/drv.ShowCmd.Run`).

I guess we currently have to, because we can't specify another output location other than `os.Stdout`?